ang/civicase/DashboardTab.html

```diff
@@ -0,0 +1,9 @@
+<div class="civicase__dashboard__tab">
+  <div class="civicase__dashboard__tab__top"></div>
+  <div class="civicase__dashboard__tab__main">
```

Comment:
Will it be better to call this `civicase__dashboard__tab__body` or `civicase__dashboard__tab__content`? 
(low priority comment)